README.md

```diff
@@ -8,8 +8,10 @@ MinIO uses [Sphinx](https://www.sphinx-doc.org/en/master/index.html) to generate
 
 - Any GNU/Linux Operating System.
```

Comment:
Probably worth dropping this line, or at least adding MacOS to it.

Reply:
macOS 12.3 (Monterey) finally stopped shipping py 2.7, so that might be a reasonable lower bound?

Reply:
Seems reasonable to me.